pkg/network/tracer/tracer_linux_test.go

```diff
@@ -576,24 +576,31 @@ func (s *TracerSuite) TestUnconnectedUDPSendIPv6() {
 	tr := setupTracer(t, cfg)
 	linkLocal, err := offsetguess.GetIPv6LinkLocalAddress()
 	require.NoError(t, err)
+	remoteAddr := linkLocal[0]
+	remoteAddr.Port = rand.Int()%5000 + 15000
 
-	remotePort := rand.Int()%5000 + 15000
-	remoteAddr := &net.UDPAddr{IP: net.ParseIP(offsetguess.InterfaceLocalMulticastIPv6), Port: remotePort}
```

Comment:
Flagging that I changed the `remoteAddr` away from multicast, this is because SOCK_RAW does not capture multicast traffic unless you add extra sockopts and flags to the pktType in the bind address [documented here](https://stackoverflow.com/q/30878554)